sqlite3.sql

```diff
@@ -0,0 +1,26 @@
+.open olm_catalog_indexes/index.db.4.6.redhat-operators
```

Comment:
See that you are commiting the bin above with the index.db. I do not think that we need to do that.

Reply:
I agree it's not great practice, but in terms of assuring user has something to try this out with it seems like a good first pass. I agree we should make a more elegant solution later.